scripts/restart.sh

```diff
@@ -0,0 +1,2 @@
+docker-compose down
+nohup docker-compose up --force-recreate --remove-orphans >/dev/null 2>&1 &
```

Comment:
1. GitHub complaining about no newline at end of file; might be best to address.
2. If you run this from the scripts directory, it won't work, but I guess the failure will be obvious.